Sources/Nimble/Adapters/NMBObjCMatcher.swift

```diff
@@ -57,9 +57,14 @@ public class NMBObjCMatcher: NSObject, NMBMatcher {
 
     public func matches(_ actualBlock: @escaping () -> NSObject?, failureMessage: FailureMessage, location: SourceLocation) -> Bool {
         let expr = Expression(expression: actualBlock, location: location)
-        let result = _match(
-            expr,
-            failureMessage)
+        let result: Bool
+        do {
+            result = try _match(expr, failureMessage)
+        } catch let error {
```

Comment:
No need for `let error` here as `error` is bound automatically

Reply:
This is just for consistency in the codebade. I prefer using just `catch { ... }` so let's replace them in a separate PR.